Core/TrackerDataManager.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  TrackerData.swift
+//  Core
+//
+//  Copyright © 2019 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+public class TrackerDataManager {
+    
+    public static let shared = TrackerDataManager()
+    
+    private(set) public var trackerData: TrackerData!
+    
+    init() {
+        reload()
+    }
+    
+    public func reload() {
+        // valid use case is we don't have downloaded data yet
+        let data = FileStore().loadAsData(forConfiguration: .trackerDataSet) ?? Self.loadEmbeddedAsData()
```

Comment:
I'd consider making this flow a bit more explicit:
- Path for `trackerDataSet` only.
- Path for fallback/error handling. 

Above line sort of mixes both and we will miss any issues around storing/reading data set file (instead we will quietly fallback to embedded data set).